composer.json

```diff
@@ -9,10 +9,13 @@
         "symfony/dotenv": "5.0.*",
         "symfony/flex": "^1.3.1",
         "symfony/framework-bundle": "5.0.*",
+        "symfony/orm-pack": "^1.0",
+        "symfony/security-bundle": "5.0.*",
         "symfony/twig-pack": "^1.0",
         "symfony/yaml": "5.0.*"
     },
     "require-dev": {
+        "doctrine/doctrine-fixtures-bundle": "^3.3"
```

Comment:
Add PHPStan? It has advantages if you add this package right now.

Reply:
I was indeed planning on adding phpstan, but this will be done via a separate issue.

Reply:
Created #26 26